data/reusables/github-actions/actions-billing.md

```diff
@@ -1 +1 @@
-{% data variables.product.prodname_actions %} usage is free for public repositories and self-hosted runners. For private repositories, each {% data variables.product.prodname_dotcom %} account receives a certain amount of free minutes and storage, depending on the product used with the account. Any usage beyond the included amounts is controlled by spending limits.
+{% data variables.product.prodname_actions %} usage is free for both public repositories and self-hosted runners. For private repositories, each {% data variables.product.prodname_dotcom %} account receives a certain amount of free minutes and storage, depending on the product used with the account. Any usage beyond the included amounts is controlled by spending limits.
```

Comment:
👋 Hi, @hwine! Thanks a lot for this suggestion; I agree that this could be misunderstood.

I think we could probably clear this up a little more, because IIUC, self-hosted runner usage is free for private repositories, but only for minutes. Storage is still counted for any uploaded workflow artifacts from self-hosted runners. The current wording might incorrectly imply that if you use self-hosted runners in a private repository, any uploaded storage usage is also free. 

```suggestion
{% data variables.product.prodname_actions %} usage is free for public repositories. For private repositories, each {% data variables.product.prodname_dotcom %} account receives a certain amount of free minutes and storage, depending on the product used with the account. Self-hosted runner usage is free, and is not counted as billable minutes. Any usage beyond the included amounts is controlled by spending limits.
```

@chrispat Could you please confirm the wording ☝️ is ok?

Reply:
One comment - I sure wouldn't know how to work it in smoothly - actions are not available for private repos on legacy plans. (I oversee a couple of those kind of orgs.)

Reply:
That wording is correct.

Reply:
> One comment - I sure wouldn't know how to work it in smoothly - actions are not available for private repos on legacy plans. (I oversee a couple of those kind of orgs.)

@hwine Thanks, but this is already mentioned in the purple product callout that is at the top of most docs pages for Actions 🙂 , e.g.

![image](https://user-images.githubusercontent.com/4434330/105122223-8cd8d100-5b21-11eb-91b5-c168a48b280a.png)

Reply:
oh, duh. I have to remember to read the top matter even when I enter via a deep link.